orchestrator.py

```diff
@@ -49,7 +49,7 @@ def attach(self):
             config_type = manager_class.config_model
             for instance_config in config_type.select().order_by(config_type.id):
                 self._load_manager_for_config(manager_class, instance_config)
-                return
+            return
```

Comment:
You can just remove the return - it's also wrong here, since this way it will only load 1 type of manager. I had it added as a debug measure, so it should be removed altogether.